controllers/apply/dashboard/lib/get-notice.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const config = require('config');
+const get = require('lodash/fp/get');
+const { oneLine } = require('common-tags');
+
+module.exports = function getNotice(locale, pendingSimple = []) {
```

Comment:
The `pendingSimple` name here is a bit confusing, could we call it `pendingApplications` or something to make it clearer what kind of array it's operating over? (I know it's Simple apps as opposed to Standard but it's not clear it's _applications_ from the name).

Reply:
Good shout, forgot to rename this.